src/flight_paths.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+"""Return a flight path between two airports."""
+from future.standard_library import install_aliases
+install_aliases()
+
+from urllib.request import urlopen
+import json
+
+
+DATA_URL = 'https://codefellows.github.io/sea-python-401d4/_downloads/cities_with_airports.json'
+
+
+def read_file(url):
+    """Read a file from the internets."""
+    response = urlopen(url)
+    html = response.read()
+    return html.decode('utf-8')
+
+
+def parse_json(data):
+    """Parse a json request."""
+    json_data = json.loads(data)
+    return json_data
```

Comment:
You probably don't need your own function for this if really all you're doing is this single thing.